syntax = "proto3";

message LeaveRoomC2S {
    
}